app/src/main/java/com/glowstudio/android/blindsjn/utils/TimeUtils.kt

```
@@ -0,0 +1,36 @@
+package com.glowstudio.android.blindsjn.utils
+
+import java.text.SimpleDateFormat
+import java.util.*
+
+object TimeUtils {
+    // String 형태의 날짜를 파싱해서 "~ 전" 표시를 돌려줍니다.
+    fun getTimeAgo(dateStr: String): String {
+        val dateFormat = SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.getDefault())
+        val date = try {
+            dateFormat.parse(dateStr)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)